components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -81,6 +81,48 @@ To access these components for an organization, select **Chef Servers > Organiza
 
 A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuration and policy distribution. A cookbook contains recipes and other files, components, or directories.
 
+The Chef Infra Server UI lets you:
+
+- List out cookbooks with the latest version
+- View different versions of a cookbook
+- View the contents of a cookbook
+- View the details of a cookbook
+
+#### List of Cookbooks
+
+In Chef Infra Server, you can view all the cookbooks of an organization. Each Cookbook has its latest version mentioned in the list.
```

Comment:
This is the default view, isn't it? 
`Each Cookbook has its latest version mentioned in the list.` is passive, please revise.